rfd/0107-abandon-active-access-request.md

```diff
@@ -0,0 +1,103 @@
+---
+authors: Jeff Pihach (jeff.pihach@goteleport.com)
+state: draft
+---
+
+# RFD 0107 - Abandon Active Access Requests
+
+## Required Approvers
+
+- Engineering: @zmb3
+
+## What
+
+Allow users who are using elevated permissions via an access request to abandon
+them ahead of their expiry once they are no longer of use.
+
+## Why
+
+By reducing the amount of time that a user has elevated permissions we also
+reduce the time and surface area to which an attack could occur in the event
+the users device becomes compromised.
+
+## Success Criteria
+
+- A user is able to abandon an active access request ahead of their expiry time
+  via the CLI or UI.
+- Creating a lock to prevent further use of the existing certificate.
+- An abandoned access request is unrecoverable and unusable.
+
+## Details
+
+### CLI
+
+A new `abandon` sub command to `tsh request` will be added. This command will
+call an GRPC method `AbandonAccessRequest` to mark the access request as
+`ABANDONED` and to set a lock on the access request.
+
+### UI
+
+When viewing your access request, in addition to the unassume option, there
```

Comment:
This is not when viewing, but it's when accessing the web UI in the elevated context provided by an access request.

Reply:
Right thanks, updated.